README.md

```diff
@@ -1,25 +1,31 @@
 # preCICE adapter for MBDyn #
 
 ## Installation ##
+No installation configured, instead include folder in PYTHONPATH
 ```
-sudo python setup.py install
+export PYTHONPATH=$PYTHONPATH:<path-to-adapter>
 ```
 
 Dependencies:
-*    MBDyn with Python interface
-*    preCICE with Python interface
+*    MBDyn with Python interface, develop branch for python 3 support
+*    preCICE and preCICE/python-bindings, version 2 or higher
 *    OpenFOAM-adapter for preCICE
 
 ## MBDyn installation ##
 ```
-wget https://www.mbdyn.org/userfiles/downloads/mbdyn-1.7.3.tar.gz
-tar -xf mbdyn-1.7.3.tar.gz
-cd mbdyn-1.7.3
-CPPFLAGS=-I/usr/include/suitesparse ./configure --enable-python=yes
-make 
-make install
+git clone https://public.gitlab.polimi.it/DAER/mbdyn.git
+cd mbdyn
+git checkout develop
+CPPFLAGS=-I/usr/include/suitesparse PYTHON_VERSION=3 ./configure --enable-python=yes
```

Comment:
I am running this case to check the workflow. I think `sh bootstrap.sh` needs to be added before this command